src/parser/jobs/rdm/modules/Swiftcast.tsx

```diff
@@ -12,6 +13,14 @@ export class Swiftcast extends CoreSwiftcast {
 	static override displayOrder = DISPLAY_ORDER.SWIFTCAST
 
 	@dependency private downtime!: Downtime
+	@dependency private actors!: Actors
+
+	// If Swiftcast and Acceleration are both up, these actions consume Acceleration first.
+	private ACCELERATION_ACTIONS: ActionKey[] = [
+		'VERTHUNDER_III',
+		'VERAERO_III',
+		'IMPACT',
+	]
```

Comment:
This has to be a class var in order to use this.data right?

Reply:
Correct, so we use the ActionKeys in a const and then map to fill, or just declare outright as a class var with `this.data` for the IDs. If you prefer the former, move the map into the body and set the class var from that, rather than running it in the function.

Reply:
I went with the class var, avoids some mapping.